### PR TITLE
update go to 1.13 & rust to 1.37 in circleci config

### PR DESCRIPTION
**Pull request by 3100**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
   contract-test:
     working_directory: /go/src/github.com/bluele/hypermint
     docker:
-      - image: rust:1.34.1-stretch
+      - image: rust:1.37.0-buster
```
Comment (bluele) [inline .circleci/config.yml]: should we upgrade rust image version?

File: README.md [hunk 2]
```diff
@@ -13,10 +13,11 @@ hypermint is a Tendermint-based blockchain that support WebAssembly smart contra
 ## Build
 
 ```
-$ export GO111MODULE=on
 $ make build
 ```
 
+If you use Golang 1.12, export `GO111MODULE=on` for modules.
```
Comment (bluele) [inline README.md]: More strictly, 1.12 or 1.11?

File: hmc/src/lib.rs [hunk 3]
```diff
@@ -66,12 +66,7 @@ extern "C" {
         ret: *mut u8,
```
Comment (bluele) [inline hmc/src/lib.rs]: Can you move this changes to other PR?

Comment (3100) [reply hmc/src/lib.rs]: Sure, I will make another PR.

File: README.md (hunk 2 above)
Comment (3100) [reply README.md]: Good catch, thanks.

File: tests/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM rust:1.34.1-stretch
+FROM rust:1.37.0-buster
```
Comment (3100) [inline tests/Dockerfile]: Without this change, `docker build` still works.
Should I remove all changes related with rust?

**Comment by bluele** [comment]
@3100 
I've pushed a new go-rust image to dockerhub.
Link is here: https://cloud.docker.com/u/hypermint/repository/docker/hypermint/go-rust/general
Could you update circleci settings with `hypermint/go-rust:v0.2.0`?

File: .circleci/config.yml (hunk 1 above)
Comment (bluele) [reply .circleci/config.yml]: NOTE: we decided to update the rust image with this PR

File: tests/Dockerfile (hunk 4 above)
Comment (bluele) [reply tests/Dockerfile]: NOTE: we decided to update the rust image with this PR

**Pull request merged**